DOCUMENTATION.md

```diff
@@ -185,6 +187,24 @@ an easy method to test on Windows / Internet Explorer. We use BrowserStack in tw
 
 Create a personal account on BrowserStack and then contact a developer to get your account associated with the CDS account.
 
+### Deployment
+
+#### How are we deploying the application?
+
+To standardize the environments between testing and production we are using [Docker](https://www.docker.com/), which allows us to create the same software conditions throughout our release pipeline. Docker allows us to create a specific environment from which the application should run in, defined through the [Dockerfile](https://github.com/cds-snc/vac-benefits-directory/blob/master/Dockerfile) inside our main repository. Docker executes this file and builds a container image, which then can be deployed on any infrastructure that supports container images (Azure, AWS, Google Cloud, etc.)
```

Comment:
"from which the application should run in" -> "from which the application should run"